### PR TITLE
Safely update balances

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #4904 

1. Decrease verbosity of known geth connection issue to INFO
1. Try to update other balances even if one fails

### CODE REVIEW

File: golem/ethereum/transactionsystem.py [hunk 1]
```diff
@@ -956,15 +958,33 @@ def _refresh_balances(self) -> None:
 
         # Sometimes web3 may throw but it's fine here, we'll just update the
         # balances next time
-        try:
+        @contextlib.contextmanager
+        def safe_update(currency):
+            print("safe update", currency)
```
Comment (maaktweluit) [inline golem/ethereum/transactionsystem.py]: why use `print` here? would be cleaner to use logger.debug i think

**Review by maaktweluit — CHANGES_REQUESTED** [review]
LGTM!

One small q, will approve on fix or reply

File: golem/ethereum/transactionsystem.py (hunk 1 above)
Comment (jiivan) [reply golem/ethereum/transactionsystem.py]: My bad. Those are my debug debris

Comment (jiivan) [reply golem/ethereum/transactionsystem.py]: Fixed

**Review by maaktweluit — APPROVED** [review]

**Pull request merged**